tests/src/test/java/alluxio/cli/fs/command/LsCommandIntegrationTest.java

```diff
@@ -366,4 +366,123 @@ public void lsWithFormatSpecifierCharacter() throws IOException, AlluxioExceptio
         STATE_FILE_IN_ALLUXIO, file.getPersistenceState());
     Assert.assertEquals(expected, mOutput.toString());
   }
+
+  /**
+   * Tests ls command with sort option.
+   */
+  @Test
+  @LocalAlluxioClusterResource.Config(
+          confParams = {PropertyKey.Name.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "false",
+                  PropertyKey.Name.SECURITY_AUTHENTICATION_TYPE, "NOSASL"})
+  public void lsWithSortByPath() throws IOException, AlluxioException {
+    FileSystemTestUtils
+            .createByteFile(mFileSystem, "/testRoot/testFileA", WriteType.MUST_CACHE, 50);
+    FileSystemTestUtils
+            .createByteFile(mFileSystem, "/testRoot/testFileZ", WriteType.MUST_CACHE, 10);
+    FileSystemTestUtils
+            .createByteFile(mFileSystem, "/testRoot/testLongFile", WriteType.MUST_CACHE, 100);
+    URIStatus[] files = new URIStatus[3];
+    files[0] = mFileSystem.getStatus(new AlluxioURI("/testRoot/testFileA"));
```

Comment:
clearer to call each file by its basename, i.e.
```java
URIStatus testFileA = mFileSystem.getStatus(new AlluxioURI("/testRoot/testFileA"));
URIStatus testFileZ = mFileSystem.getStatus(new AlluxioURI("/testRoot/testFileZ"));
URIStatus testLongFile = mFileSystem.getStatus(new AlluxioURI("/testRoot/testLongFile"));
```

Reply:
Agree.